draft-ietf-core-coap-tcp-tls.md

```diff
@@ -138,8 +138,9 @@ reliable delivery, simple congestion control, and flow control.
 Some environments benefit from the availability of CoAP carried over reliable
 transports such as TCP or TLS. This document outlines the changes required to use
 CoAP over TCP, TLS, and WebSockets transports. It also formally updates RFC 7641
-for use with these transports, and RFC 6455 to extend the well-known
-URI mechanism (RFC 5785) to the ws and wss URI schemes.
+for use with these transports, RFC 6455 to extend the well-known
+URI mechanism (RFC 5785) to the ws and wss URI schemes, and RFC 7959 to
+to enable the use of larger messages over a reliable transport.
 

```

Comment:
s/to to/to/

(The sequence 7641 6455 7959 is slightly weird; I had the external housekeeping for 6455 last, maybe we can insert 7959 between 7641 and 7959.)